extensions/typescript-language-features/src/languageFeatures/tsconfig.ts

```diff
@@ -122,5 +175,15 @@ export function register() {
 		languages.map(language =>
 			patterns.map((pattern): vscode.DocumentFilter => ({ language, pattern }))));
 
-	return vscode.languages.registerDocumentLinkProvider(selector, new TsconfigLinkProvider());
+	return vscode.Disposable.from(
+		vscode.commands.registerCommand(openExtendsLinkCommandId, async ({ resourceUri, extendsValue, }: OpenExtendsLinkCommandArgs) => {
+			const tsconfigPath = await getTsconfigPath(Utils.dirname(resourceUri), extendsValue);
+			if (tsconfigPath === undefined) {
+				vscode.window.showErrorMessage(localize('openTsconfigExtendsModuleFail', "Failed to resolve {0} as module", extendsValue));
+				return;
+			}
+			await vscode.workspace.openTextDocument(tsconfigPath);
```

Comment:
`openTextDocument` opens the text document but you then need to call `showTextDocument` to show it

Reply:
Also, you may need to use the `vscode.open` command instead of `openTextDocument` + `showTextDocument`. I believe the command handles the case where the file does not exist and lets users create it if they want (but please test to confirm this)

Reply:
> `openTextDocument` opens the text document but you then need to call `showTextDocument` to show it

I don't really understand the difference between these, though, even my friend [tried](https://github.com/zardoy/vscode-framework/issues/9#issuecomment-951319097) to explaind it. Nothing worked here except the `vscode.open` command.

However the latter just shows error notification instead of modal, but with desired `create file` button!